vite.config.js

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
```

Comment:
should be: 

```javascript

import { defineConfig } from 'vite';
import react from '@vitejs/plugin-react';

export default defineConfig(() => {
  return {
    server: {
      open: true,
      port: 3000
    },
    build: {
      outDir: 'build',
    },
    plugins: [react()],
  };
});
```
because we want the build folder to come out
we also want to keep the port the same 
src: https://www.robinwieruch.de/vite-create-react-app/

Reply:
@ta0a2000t changed